src/server/scripts/Northrend/VioletHold/boss_erekem.cpp

```diff
@@ -49,47 +61,57 @@ class boss_erekem : public CreatureScript
     {
         boss_erekemAI(Creature* creature) : ScriptedAI(creature)
         {
-            Initialize();
```

Comment:
by removing the call to Initialize() you left all the class fields uninitialized, triggering static analysis issues